build/upload_tarballs.jl

```diff
@@ -92,15 +64,13 @@ if abspath(PROGRAM_FILE) == @__FILE__
     # check that contents are tarballs with correct filename
     all(t -> !isnothing(match(TARBALL_REGEX, t)), readdir(TARBALL_DIR))
 
-    pkg_http_url = parse_git_remote_url(PKG_URL)
-    artifact_url = "$(pkg_http_url)/releases/download/$TAG"
+    all(readdir(TARBALL_DIR)) do t
+        contains(t, "ray_julia.$TAG") || error("Stale tarball found: $t")
+    end
```

Comment:
Using `all` here is a bit strange due to having an internal error. A `for` or this seems better:
```suggestion
    stale = filter(!startswith("ray_julia.$TAG"), readdir("tarballs"))
    isempty(stale) || error("Stale tarball(s) found:\n$(join(stale, '\n'))")
```